packages/lesswrong/server/inactiveUserSurveyCron.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { addCronJob } from './cronUtil';
+import { wrapAndSendEmail } from './emails/renderEmail';
+import './emailComponents/EmailInactiveUserSurvey';
+import { Components, Globals } from './vulcan-lib';
+import { loggerConstructor } from '../lib/utils/logging';
+import { UsersRepo } from './repos';
+import Users from '../lib/vulcan-users';
+import { isEAForum } from '../lib/instanceSettings';
+
+
+/**
+ * Sends emails to inactive users with a link to a feedback survey
+ */
+const sendInactiveUserSurveyEmails = async () => {
+  if (!isEAForum) return
+  
+  const logger = loggerConstructor(`cron-sendInactiveUserSurveyEmails`)
+  
+  // Get up to 30 inactive users to email
+  const users = await new UsersRepo().getInactiveUsersToEmail(30)
```

Comment:
It looks like each day we get about 8 users that would qualify for this email. I think many of those are uncaught spam. I'm not sure why they are uncaught since I thought our forum facilitators looked at all new users who write something in their profile.

In any case, I would prefer not to email spammers but I think it's better to do that than to miss real users. I'm limiting it to 30 per day for now in case there are bugs, and there is no urgency to sending this email. I'll increase this limit later if there are no issues.